strategies/failed/src/main/java/org/arquillian/smart/testing/strategies/failed/TestResultsFilter.java

```diff
@@ -0,0 +1,34 @@
+package org.arquillian.smart.testing.strategies.failed;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import org.arquillian.smart.testing.TestSelection;
+import org.arquillian.smart.testing.spi.TestResult;
+
+public class TestResultsFilter {
```

Comment:
This can be package-private. We can make it public when needed

Reply:
good catch 👍